.gitpod.yml

```diff
@@ -1,8 +1,8 @@
 ports:
 - port: 3000
 tasks:
-- init: yarn install
-  command: cd examples/browser && yarn start --hostname 0.0.0.0 ../..
+- init: nvm install 10 && nvm use 10 && yarn
+  command: yarn --cwd examples/browser start ../..
```

Comment:
This will eventually fix https://github.com/theia-ide/theia/issues/4818 too but you have to restore `--hostname 0.0.0.0`, don't you?

Reply:
Gitpod is capable to auto expose localhost now, providing hostname is redundant.